doc/source/commands/flight/cooked.rst

```diff
@@ -61,7 +67,62 @@ If you ``LOCK`` the ``THROTTLE`` or ``STEERING``, be aware that this prevents th
 
 When the program ends, these automatically unlock as well, which means that to control a craft you must make sure the program doesn't end. The moment it ends it lets go of the controls.
 
+Tuning cooked steering
+----------------------
+
+While cooked steering tries to come balanced to perform decently without user
+interaction, there are some instances where you may need to help tune the
+behavior.  There are two parts of the steering calculation that can be tuned.
```

Comment:
I'll give it a try tonight.  I'll pull your PR to my local and then make a few more edits to push back at you for your approval.